bundles/AdminBundle/Resources/public/js/pimcore/document/document.js

```diff
@@ -77,7 +77,7 @@ pimcore.document.document = Class.create(pimcore.element.abstract, {
         tabPanel.setActiveItem(tabId);
     },
 
-    save: function (task, only, callback) {
+    save: function (task, only, callback, successCallback) {
```

Comment:
update page_snippet::save to pass successCallback to $super:
https://github.com/pimcore/pimcore/blob/b82377b72d2b1d9deb4f1db8c9e9a74b72ff911e/bundles/AdminBundle/Resources/public/js/pimcore/document/page_snippet.js#L460

Reply:
Good catch.